lib/cmwx1/cmwx1.py

```diff
@@ -0,0 +1,367 @@
+# This file is part of the cmwx1 module.
+#
+# Copyright (c) 2013-2021 Ibrahim Abdelkader <iabdalkader@openmv.io>
+# Copyright (c) 2013-2021 Sebastian Romero <s.romero@arduino.cc>
+# Copyright (c) 2021 Arduino SA
+#
+# This work is licensed under the MIT license, see the file LICENSE for details.
+#
+# CMWX1ZZABZ-093 driver for Arduino boards.
+# Note this module runs a stock or a custom Arduino firmware.
+
+from time import sleep_ms
+from time import ticks_ms
+from machine import UART
+from machine import Pin
+from micropython import const
+
+
+class LoraError(Exception):
+    pass
+
+
+class LoraErrorTimeout(LoraError):
+    pass
+
+
+class LoraErrorParam(LoraError):
+    pass
+
+
+class LoraErrorBusy(LoraError):
+    pass
+
+
+class LoraErrorOverflow(LoraError):
+    pass
+
+
+class LoraErrorNoNetwork(LoraError):
+    pass
+
+
+class LoraErrorRX(LoraError):
+    pass
+
+
+class LoraErrorUnknown(LoraError):
+    pass
+
+
+class Lora:
+    MODE_ABP = const(0)
+    MODE_OTAA = const(1)
+
+    CLASS_A = const("A")
+    CLASS_B = const("B")
+    CLASS_C = const("C")
+
+    BAND_AS923 = const(0)
+    BAND_AU915 = const(1)
+    BAND_EU868 = const(5)
+    BAND_KR920 = const(6)
+    BAND_IN865 = const(7)
+    BAND_US915 = const(8)
+    BAND_US915_HYBRID = const(9)
+
+    RF_MODE_RFO = const(0)
+    RF_MODE_PABOOST = const(1)
+
+    LoraErrors = {
+        "": LoraErrorTimeout,  # empty buffer
+        "+ERR": LoraError,
+        "+ERR_PARAM": LoraErrorParam,
+        "+ERR_BUSY": LoraErrorBusy,
+        "+ERR_PARAM_OVERFLOW": LoraErrorOverflow,
+        "+ERR_NO_NETWORK": LoraErrorNoNetwork,
+        "+ERR_RX": LoraErrorRX,
+        "+ERR_UNKNOWN": LoraErrorUnknown,
+    }
+
+    def __init__(
+        self,
+        uart=None,
+        rst_pin=None,
+        boot_pin=None,
+        band=5,
```

Comment:
```suggestion
        band=None,
```

Reply:
I reverted the constant, was removed to fix the linter.